### PR TITLE
Remove S3

### PR DESCRIPTION
**Pull request by dpgraham**
Take out AWS and S3 because we're using bintray now.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,4 +7,3 @@ after_success:
   - npm run coverage
 script:
   - npm test
-  - npm run e2e-test
```
Comment (mykola-mokhnach) [inline .travis.yml]: so there are no more e2e test now?

**Comment by imurchie** [comment]
The dependencies can be pared down a little more. The following works:
```diff
diff --git a/package.json b/package.json
index e48d127..5e7a0ea 100644
--- a/package.json
+++ b/package.json
@@ -31,8 +31,7 @@
     "bluebird": "^3.5.1",
     "lodash": "^4.17.5",
     "sinon": "^6.0.0",
-    "source-map-support": "^0.5.4",
-    "uuid": "^3.0.1"
+    "source-map-support": "^0.5.4"
   },
   "scripts": {
     "prepublish": "gulp prepublish",
@@ -50,11 +49,9 @@
     "test"
   ],
   "devDependencies": {
-    "appium": "^1.7.2",
     "appium-gulp-plugins": "^2.2.2",
     "babel-eslint": "^7.1.1",
     "chai": "^4.1.2",
-    "chai-as-promised": "^7.1.1",
     "colors": "^1.1.2",
     "eslint": "^3.10.2",
     "eslint-config-appium": "^2.0.1",
@@ -64,10 +61,7 @@
     "eslint-plugin-promise": "^3.3.1",
     "gulp": "^3.8.11",
     "mocha": "^5.0.5",
-    "mock-fs": "^4.4.1",
-    "pre-commit": "^1.2.2",
-    "request-promise": "^4.2.0",
-    "wd": "^1.2.0"
+    "pre-commit": "^1.2.2"
   },
   "greenkeeper": {
     "ignore": [
```

File: .travis.yml (hunk 1 above)
Comment (dpgraham) [reply .travis.yml]: @mykola-mokhnach That's right. The only e2e test was the S3 test.

**Comment by dpgraham** [comment]
Fixes pushed

**Pull request merged**